### ISSUE TITLE
lbryum commands returns are  inconsistent and undocumented 

### ISSUE THREAD

**Issue by kaykurokawa** [lbryio/lbryum#139]
Commands, specifically resolution related commands, in lbryum/commands.py return inconsistent outputs and are undocumented if they do. The output seems espeically inconsistent in the case of errors, i.e. claim is not found.

We need to make them consistent, write tests for them (in lbry-in-a-box lbryum testing), and provide some documentation for the commands.

**Comment by tzarebczan** [comment]
Would this be an example of one?

`2017-09-11 17:07:39,835 INFO     lbrynet.core.Wallet:823: Resolving lbry://@headblobzzzzz/win101607englishx64
2017-09-11 17:07:51,857 ERROR    lbryum.commands:1145: No valid claims for this name for this channel
2017-09-11 17:07:51,857 ERROR    lbryum.commands:1145: No valid claims for this name for this channel`

**Comment by kauffj** [comment]
This is probably out of scope, but it would be fantastic if we began returning standardized error codes so that applications can control the way they choose to display or handle a certain failure.

Already the app has to occasionally result to inspection of messages.

**Comment by eukreign** [comment]
Once lbryum is merged into lbry it will be upto lbry command to return proper responses.
